.drone.env

```diff
@@ -1,3 +1,3 @@
 # The test runner source for UI tests
-WEB_COMMITID=67ced24b3a6f06b767ff8111537b4a0b85c40993
-WEB_BRANCH=changeRespCode
+WEB_COMMITID=6514050264308fc630abf0fbf5b4dcf728ab4f44
+WEB_BRANCH=master
```

Comment:
@2403905 @ScharfViktor I suppose this goes back to master eventually?

Reply:
Yes, after merging this one https://github.com/owncloud/web/pull/9934

Reply:
PR merged